addons/blocks2image/userscript.js

```diff
@@ -90,6 +90,14 @@ export default async function ({ addon, console, msg }) {
             exportBlock(true);
           },
           separator: false,
+        },
+        {
+          enabled: true,
+          text: msg("copy_all_to_clipboard"),
+          callback: () => {
+            exportBlock(true, "", true);
```

Comment:
This isn't a big deal but  the empty argument could be removed if the 2nd and 3rd were swapped everywhere.